### PR TITLE
OSSMDOC-520: Update traffic management headings and reorganize assembly.

### PR DESCRIPTION
**Pull request by JStickler**
Version(s): 4.6 - 4.12

Issue: [OSSMDOC-520](https://issues.redhat.com/browse/OSSMDOC-520)

This is all content that has been previously reviewed and published.  But published in long, disorganized topics, with headings that were less than helpful for navigation. I've broken things out into smaller topics and rearranged things so that it's easier to find the sections for each of the types of service mesh resources (gateways, virtualServices, serviceEntries, etc.).  Previously the topic on Gateways was buried so deep in the content that the heading did not appear in the right hand navigation on docs.openshift.  Making the gateways topic visible is preliminary work to being able to document gateway injection as part of OSSMDOC-702.

Link to docs preview:
V2 docs http://file.bos.redhat.com/jstickle/OSSMDOC-520/service_mesh/v2x/ossm-traffic-manage.html

V1 docs (Out of support, but uses the same files and thus needed to be updated with the new modules to not break the build)
http://file.bos.redhat.com/jstickle/OSSMDOC-520/service_mesh/v1x/ossm-traffic-manage.html 

QE review: matejnesuta (comments via Slack)
Peer review:  mjpytlak
Peer reviewers - I have fixed 4 Vale errors, reviewed all the Vale warnings (they were all correct spellings that haven't been added to the known words list).  Addressing the 100+ suggestions is outside of the scope of what I need to do in this PR.

### CODE REVIEW

**Comment by prachiyadav** [comment]
@JStickler , I compared it with https://docs.openshift.com/container-platform/4.11/service_mesh/v2x/ossm-traffic-manage.html .... I can not see the difference what exactly updated in new doc  http://file.bos.redhat.com/jstickle/OSSMDOC-520/service_mesh/v2x/ossm-traffic-manage.html  , could you please explain to me what exactly i need to verify .

**Comment by JStickler** [comment]
@prachiyadav The original organization of the Traffic Management section buried some of the headings so that they did not appear in the right hand table of content, which made it harder to find the topics.  I rearranged this section and changed some headings to make it easier to find topics. What I really need is someone to skim through the content and validate that my copy/pasting into new sections didn't miss any content or copy something to the wrong sections. 

Currently published headings for Traffic Management
CurrentTrafficManagement


New headings (in this PR) for Traffic Management
NewTrafficManagement

**Review by mjpytlak — COMMENTED** [review]
@JStickler Overall the reorg is good and the new content is well written. Left a few comments behind.

File: modules/ossm-routing-virtual-service.adoc [hunk 1]
```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v1x/ossm-traffic-manage.adoc
+// * service_mesh/v2x/ossm-traffic-manage.adoc
+
+:_content-type: PROCEDURE
+
+[id="ossm-routing-virtual-services_{context}"]
+= Using VirtualServices
```
[the remark this answers is not in the document]
Comment (mjpytlak) [reply modules/ossm-routing-virtual-service.adoc:9]: @JStickler Given the nature of this PR (a reorg of existing approved content to improve content findability), I believe that breaking up the content to adhere to mod doc guidelines is beyond the scope of this PR. Further, I do understand these improvements do gate other feature work. Per our discussion, please do follow up in another PR to break up this content.

File: modules/ossm-routing-ingress.adoc [hunk 2]
```diff
@@ -28,7 +28,10 @@ ifdef::openshift-enterprise[]
 You can access the gateway using the service's xref:../../networking/configuring-node-port-service-range.adoc[node port].
 endif::[]
 
+////
+TO DO - remove XREF in this module.
 Determine the ingress according to your environment. For an environment with load balancer support, xref:../../service_mesh/v2x/ossm-traffic-manage.adoc#ossm-routing-config-ig-lb_routing-traffic[Determining ingress ports with a load balancer]. For an environment without load balancer support, xref:../../service_mesh/v2x/ossm-traffic-manage.adoc#ossm-routing-config-ig-no-lb_routing-traffic[Determining ingress ports without a load balancer]. After you have determined the ingress ports, see xref:../../service_mesh/v2x/ossm-traffic-manage.adoc#ossm-routing-gateways_routing-traffic[Configuring ingress using a gateway] to complete your configuration.
+////
```
[the remark this answers is not in the document]
Comment (mjpytlak) [reply modules/ossm-routing-ingress.adoc:34]: Had a chance to chat with @JStickler and as suspected, there is a separate Jira for removing xrefs in modules. Given the nature of this PR (a reorg of existing approved content to improve content findability), addressing the xrefs is beyond the scope of this PR.

[the remark this answers is not in the document]
Comment (JStickler) [reply modules/ossm-routing-ingress.adoc:34]: [OSSMDOC-593 ](OSSMDOC-593)tracks XREFs that need to be removed from modules.

File: modules/ossm-routing-virtual-service.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (JStickler) [reply modules/ossm-routing-virtual-service.adoc:9]: I'll break this up in the next PR, for [OSSMDOC-702](https://issues.redhat.com/browse/OSSMDOC-702).

**Pull request merged**